index.bs

```diff
@@ -811,7 +810,7 @@ in a [=/Realm=] |realm|, perform the following steps:
       1. [=file entry/lock/release|Release the lock=] on |stream|.[=FileSystemWritableFileStream/[[file]]=].
       1. [=/Resolve=] |closeResult| with `undefined`.
    1. Return |closeResult|.
-1. Let |abortAlgorithm| be the following step:
+1. Let |abortAlgorithm| be these steps:
       1. [=file entry/lock/release|Release the lock=] on |stream|.[=FileSystemWritableFileStream/[[file]]=].
```

Comment:
```suggestion
1. Let |abortAlgorithm| be this step: [=file entry/lock/release|release the lock=] on 
   |stream|.[=FileSystemWritableFileStream/[[file]]=].
```
We generally don't use dots to separate fields. It's almost always `'s `, but I see this is done here more often.